docs/getting_started.md

```diff
@@ -81,6 +81,37 @@ done
 
 ## Register BYOH host to management cluster
 
+
+### Generating the Bootstrap Kubeconfig file
+Get the APIServer and Certificate Authority Data info
+
+```shell
+$ APISERVER=$(kubectl config view -ojsonpath='{.clusters[0].cluster.server}')
+$ CA_CERT=$(kubectl config view --flatten -ojsonpath='{.clusters[0].cluster.certificate-authority-data}')
```

Comment:
If we remove the prefix `$` then it will be easy to copy & paste
```suggestion
APISERVER=$(kubectl config view -ojsonpath='{.clusters[0].cluster.server}')
CA_CERT=$(kubectl config view --flatten -ojsonpath='{.clusters[0].cluster.certificate-authority-data}')
```